skimage/feature/tests/test_orb.py

```diff
@@ -78,27 +72,46 @@ def test_keypoints_orb_less_than_desired_no_of_keypoints():
-                                [ True, False,  True, False, False, False,  True,  True, False, False]], dtype=bool)
-
+    exp_descriptors = np.array([[False,  True,  True,  True, False,  True,
+                                 False,  True, False,  True],
+                                [ True,  True,  True, False, False,  True,
+                                 False, False,  True,  True],
+                                [ True, False,  True,  True, False, False,
+                                  True,  True, False, False],
+                                [False, False, False, False, False, False,
+                                 False, False,  True, False],
+                                [False,  True, False, False, False, False,
+                                 False, False,  True, False],
+                                [ True,  True, False,  True,  True,  True,
+                                 False, False,  True,  True],
+                                [ True,  True, False,  True, False, False,
+                                  True, False,  True,  True],
+                                [False, False,  True, False,  True, False,
+                                 False,  True,  True, False],
+                                [ True, False, False, False,  True, False,
+                                 False, False, False,  True],
+                                [False,  True,  True,  True,  True,  True,
+                                  True,  True,  True,  True],
+                                [ True,  True, False,  True, False,  True,
+                                 False, False,  True,  True],
+                                [ True,  True,  True, False, False, False,
+                                  True,  True,  True, False],
+                                [ True,  True,  True,  True,  True,  True,
+                                 False, False, False, False],
+                                [ True,  True,  True, False,  True,  True,
+                                  True,  True, False, False],
+                                [ True,  True, False, False,  True, False,
+                                 False,  True, False,  True],
+                                [ True,  True, False, False, False, False,
+                                  True, False, False,  True],
+                                [False, False, False, False,  True,  True,
+                                  True, False,  True, False],
+                                [False, False, False, False,  True,  True,
+                                  True, False, False,  True],
+                                [False, False, False, False, False,  True,
+                                  True, False,  True,  True],
+                                [False, False, False, False,  True, False,
+                                  True, False,  True,  True]], dtype=bool)
```

Comment:
This can probably fit into a single line by replacing True/False with 1/0. The "dtype=bool" argument will automatically convert correctly to True/False, so the statements are equivalent, but one is much more compact! =)